frontend/automation/support/pages/create-wallet.ts

```diff
@@ -0,0 +1,47 @@
+import { expect, Locator, Page } from '@playwright/test'
+
+export class CreateWallet {
```

Comment:
> no need to create class, as we won't create different instances of it.

So you guys have evidently had a discussion about the appropriateness of the page object model and I am late to the party. Is there anything I can go and read? Because as far as I am concerned it seems very applicable here. 

I have a few reasons for feeling tests should be written this way. In my experience any tests written in the currently existing fashion will not be particularly maintainable given that there is functionality hand cranked in multiple places along with repeated locators, not to mention it is a little tough for me (especially as I am new) to understand what is going on. Also as you mentioned about the code increase- more code != more maintenance in this instance. If it is a framework we want to upscale then I think this makes it easier to scale out and maintain since there is a single source of truth. Would be good to have a discussion so I can understand the team's approach better :) @ValentinTrinque @dexturr @radoslaw-szpiech

Reply:
I think I was misunderstood a little :D Let me rephrase that.
The talk we had was a year ago, I don't think we have any archive of that on Slack. But... it's history :D We don't need to strictly keep to what we said a year ago, also we don't need to strictly keep to any approaches, e.g. POM).
The page objects you created are fine, there's a lot of reusability here. They can be used in other tests files as well, so GJ! What I said about not creating unnecessary functions was more like general advice for the future to not fall into trap of creating separate additional functions, that are handling some simple action, and they are not re-used everywhere. 
The thing about class - no need for using class, as we probably won't create different instances of it, with different parameters or keeping it's state, so it just adds more code and no real value. I will create a commit to show what I meant :)

Reply:
@dalebennett1992 https://github.com/vegaprotocol/vegawallet-desktop/pull/616 take a look here

Reply:
Merged. :) Classes are just more than what we really need here

Reply:
yeah I left a comment, essentially it is achieving the same thing tbh but if that is what you prefer then I am pretty easy about it. Thanks